morpheus/_lib/src/messages/memory/response_memory_probs.cpp

```diff
@@ -36,10 +36,10 @@ ResponseMemoryProbs::ResponseMemoryProbs(size_t count, TensorObject probs) : Res
     this->tensors["probs"] = std::move(probs);
 }
 
-ResponseMemoryProbs::ResponseMemoryProbs(ResponseMemory &&other) : ResponseMemory(other.count, std::move(other.tensors))
+ResponseMemoryProbs::ResponseMemoryProbs(size_t count, tensor_map_t &&tensors) :
+  ResponseMemory(count, std::move(tensors))
```

Comment:
```suggestion
ResponseMemory(count, std::move(tensors))
```

Reply:
clang-format didn't like this change. I had to revert it.

Reply:
Oh I see my mistake. Nevermind.